java/client/src/main/java/glide/api/models/BaseTransaction.java

```diff
@@ -2842,6 +2843,22 @@ public T geopos(@NonNull String key, @NonNull String[] members) {
         return getThis();
     }
 
+    /**
+     * Returns the bit value at <code>offset</code> in the string value stored at <code>key</code>.
+     *
+     * @see <a href="https://redis.io/commands/getbit/">redis.io</a> for details.
```

Comment:
Update to point to Valkey docs.

Reply:
To keep it consistent we decided to keep it as redis for now as there are some pages missing for valkey. Reference: https://github.com/aws/glide-for-redis/pull/1384#discussion_r1592897061